zookeeper-server/src/main/java/org/apache/zookeeper/server/ZooKeeperServer.java

```diff
@@ -704,12 +704,16 @@ protected void startRequestThrottler() {
 
     }
 
+
+    /** the request processor in here is not working because the observer, leader and follower have fished by themselves.
+     * so remove the operation in this function
+     */
     protected void setupRequestProcessors() {
-        RequestProcessor finalProcessor = new FinalRequestProcessor(this);
-        RequestProcessor syncProcessor = new SyncRequestProcessor(this, finalProcessor);
-        ((SyncRequestProcessor) syncProcessor).start();
-        firstProcessor = new PrepRequestProcessor(this, syncProcessor);
-        ((PrepRequestProcessor) firstProcessor).start();
+//        RequestProcessor finalProcessor = new FinalRequestProcessor(this);
+//        RequestProcessor syncProcessor = new SyncRequestProcessor(this, finalProcessor);
+//        ((SyncRequestProcessor) syncProcessor).start();
+//        firstProcessor = new PrepRequestProcessor(this, syncProcessor);
+//        ((PrepRequestProcessor) firstProcessor).start();
     }
```

Comment:
-1. Without this method, the standalone server cannot work to process the client's requests.